packages/adyen-salesforce-pwa/lib/api/models/basketService.js

```diff
@@ -36,6 +37,48 @@ export class BasketService {
         this.res.locals.adyen.basket = updatedBasket
     }
 
+    /**
+     * Removes any existing temporary baskets for the current customer.
+     */
+    async removeExistingTemporaryBaskets() {
```

Comment:
Could we address this as discussed [here](https://github.com/Adyen/adyen-salesforce-headless-commerce-pwa/pull/205#pullrequestreview-3482250790)